src/component/RecipeFormModal.tsx

```diff
@@ -103,13 +153,15 @@ function RecipeFormModal({ onClose }) {
     };
   }, []);
 
-  const recipeData = Object.keys(recipeFormState)
+  const recipeData = (Object.keys(recipeFormState) as (keyof RecipeFormState)[])
     .filter((key) => key !== "ingredients")
     .map((key) => {
       const labelCheck = ["cooking_time", "servings"].includes(key);
       return (
         <div key={key}>
-          <label>{labelText[key]}</label>
+          <label>
+            {labelText[key as keyof Omit<RecipeFormState, "ingredients">]}
```

Comment:
I think you can avoid this assertion by defining above `type LabelText = Omit<RecipeFormState, "ingredients">`, so that TypeScript understands that `LabelText` and `RecipeFormState` have some similar properties.